labware-library/src/analytics/fullstory.ts

```diff
@@ -1,31 +1,33 @@
-// @flow
 import uniq from 'lodash/uniq'
 
-const FULLSTORY_NAMESPACE = 'FS'
-const FULLSTORY_ORG = process.env.OT_LL_FULLSTORY_ORG
 const LL_VERSION = process.env.OT_LL_VERSION
-const LL_BUILD_DATE = new Date((process.env.OT_LL_BUILD_DATE: any))
+const LL_BUILD_DATE = new Date(process.env.OT_LL_BUILD_DATE as any)
 
 const _getFullstory = (): Object | null => {
   const namespace = global._fs_namespace
-  const fs = namespace ? global[namespace] : null
+  // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
+  const fs = namespace ? (global as any)[namespace] : null
+  // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
   return fs || null
 }
 
-export const shutdownFullstory = () => {
+export const shutdownFullstory = (): void => {
   console.debug('shutting down Fullstory')
-  const fs = _getFullstory()
-  if (fs && fs.shutdown) {
+  const fs: any = _getFullstory()
+  // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
+  if (fs?.shutdown && typeof fs.shutdown === 'function') {
     fs.shutdown()
   }
-  if (global._fs_namespace && global[global._fs_namespace]) {
-    delete global[global._fs_namespace]
+  // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
+  if (global._fs_namespace && (global as any)[global._fs_namespace]) {
+    // eslint-disable-next-line @typescript-eslint/no-dynamic-delete
+    delete (global as any)[(global as any)._fs_namespace]
   }
 }
 
 export const inferFsKeyWithSuffix = (
   key: string,
-  value: mixed
+  value: any
```

Comment:
If this was `mixed`, can it be `unknown`?

```suggestion
  value: unknown
```